include/libkrun.h

```diff
@@ -265,7 +265,109 @@ int32_t krun_add_virtiofs2(uint32_t ctx_id,
                            const char *c_path,
                            uint64_t shm_size);
 
+/* Send the VFKIT magic after establishing the connection,
+   as required by gvproxy in vfkit mode. */
+#define NET_FLAG_VFKIT 1 << 0
+
+/* Taken from uapi/linux/virtio_net.h */
+#define NET_VFLAG_CSUM 1 << 0
```

Comment:
If we go with 2 flag arguments let's make the constants and the flags names more clear using "features" and "flags".

```
#define NET_FEATURE_CSUM 1<<0
...
 *  "features"  - virtio-net features for the network interface.
 *  "flags"     - generic flags for the network interface.
```

Switching the order because the virtio features are more important and flags are usually the last argument.

Reply:
Sounds reasonable to me, I've just changed it.